install/amazon_rds/00_overview.mdx

```diff
@@ -31,6 +34,20 @@ ECS, or with Docker.
   * The collector instance needs to be able to connect to the server directly,
     bypass any connection poolers like PgBouncer
 
+Here is an overview diagram for how the pganalyze collector will be placed
+within your existing setup.
+
+<ImgCollectorAwsArchitecture />
+
+Collector can be installed in a different AWS account from the one hosting the
```

Comment:
I guess this is somewhat subjective, but I think this would work better with the definite article:
```suggestion
The pganalyze collector can be installed in a different AWS account from the one hosting the
```
or just
```suggestion
The collector can be installed in a different AWS account from the one hosting the
```